project-base/app/tests/App/Smoke/Http/RouteConfigCustomization.php

```diff
@@ -175,7 +174,7 @@ private function configureAdminRoutes(RouteConfigCustomizer $routeConfigCustomiz
             ->customize(function (RouteConfig $config, RouteInfo $info) {
```

Comment:
Reviewing this I came up with an interesting idea, given that you can use any admin on line 174, we could use Smoke tests also for testing admin rights using differently set admins.